README.md

```diff
@@ -104,9 +113,28 @@ testRule({
 });
 ```
 
-## Schema properties
+### `testInvalidRuleConfigs`
+
+The `testInvalidRuleConfigs` function enables you to test invalid configs for a rule.
+
+For example:
+
+```js
+testInvalidRuleConfigs({
+  plugins: ["."],
+  ruleName,
 
-See the [type definitions](index.d.ts).
+  configs: [
```

Comment:
```suggestion
  reject: [
```

Shall we use `reject` for consistency with the other schema to clarify that we expect these configs to be rejected?

On that subject, do you think there's value in adding an `accept` array too, i.e. it doesn't expect any `invalidOption` warnings for the specified configs? It's possible to test multiple accepted configs by creating `testRule`s, but it's a bit long-winded:

```js
testRule({
  plugins: ["."],
  ruleName,
  config: ["valid"],

  accept: [
    {
      code: ".class {}", // this is kind of redundant
    },
  ]
)}

testRule({
  plugins: ["."],
  ruleName,
  config: ["alsovalid"],

  accept: [
    {
      code: ".class {}", // this is kind of redundant
    },
  ]
)}
```

And so on.

Vs:

```js
testRuleConfigs({
  plugins: ["."],
  ruleName,

  accept: [
    {
      config: "valid"
    },
    {
      config: "alsovalid"
    },
  ],

  reject: [
    {
      config: "invalid"
    },
    {
      config: [/invalid/],
      description: "regex is not allowed"
    }
  ]
});
```

There's redundancy either way, as a `testRule` also _indirectly_ checks the validity of configs. So, there's no obvious right approach here. The latter approach could be beneficial when an author wants to test a rule that expects objects of a certain structure, e.g. [scales/font-sizes](https://github.com/jeddy3/stylelint-scales/tree/main/lib/rules/font-sizes#options). Or we keep it simple and stick with `testInvalidRuleConfigs` (with `reject` instead of `configs`).

Reply:
Thanks for the feedback. Your suggested wording seems more intuitive to me. I'll change it soon. 👍🏼

Reply:
Fixed! b77ecc8